File: Aide.ClientApp/src/app/models/claim-probatory-document.ts
import { ProbatoryDocument } from './probatory-document';
import { Media } from './media';

export interface ClaimProbatoryDocument {
    id: number;
    claimId: number;
    claimItemId?: number;
    probatoryDocumentId: number;
    sortPriority: number;
    groupId: number;
    dateCreated: string;
    dateModified: string;

    probatoryDocument?: ProbatoryDocument;
    media?: Media;
}

File: Aide.Claims.Domain/Objects/ClaimProbatoryDocument.cs
using System;

namespace Aide.Claims.Domain.Objects
{
    public class ClaimProbatoryDocument
    {
        public int Id { get; set; }
        public int ClaimId { get; set; }
        public int? ClaimItemId { get; set; }
        public int ProbatoryDocumentId { get; set; }
        public int SortPriority { get; set; }
        public int GroupId { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime DateModified { get; set; }

        public ProbatoryDocument ProbatoryDocument { get; set; }
        public Media Media { get; set; }
    }
}
